src/dsl/validation.rs

```diff
@@ -461,15 +466,15 @@ mod tests {
         let engine = Engine::new();
 
         let json_value: serde_json::Value =
-            serde_yaml::from_str(&input).expect("Unable to parse yaml");
+            serde_yaml::from_str(input).expect("Unable to parse yaml");
 
-        let deserialization_result = serde_yaml::from_str::<Check>(&input);
+        let deserialization_result = serde_yaml::from_str::<Check>(input);
 
         let json_schema = get_json_schema();
         let validation_result = validate(&json_value, "156F64", &json_schema, &engine);
 
-        assert_eq!(validation_result.is_ok(), true);
-        assert_eq!(deserialization_result.is_ok(), true);
+        assert!(validation_result.is_ok());
+        assert!(deserialization_result.is_ok());
```

Comment:
Sorry, Github ate my comment. Would you add metadata to this test or write another one with metadata and use this set of assertions? I'd like to see if also the deserialization into `Check` is fine as well

Reply:
I ended up in adding metadata to the `Check` struct and adding the deserialization result assertions in the `validate_ok_check` test

Reply:
Perfect, thanks :+1: